Dockerfile

```diff
@@ -10,4 +8,4 @@ COPY target/universal/stage/ /workdir/
 RUN chmod +x /workdir/bin/codacy-metrics-detekt
```

Comment:
It is very likely that the executable has already executable permissions.
You can try remove this layer.
```suggestion
```

Reply:
It fails with a lack of permissions without that line.